kubectl-pg/README.md

```diff
@@ -0,0 +1,53 @@
+# kubectl-pg
+
+### Installtion of kubectl pg plugin
+
+Clone the kubectl pg plugin and build from the source using ```go install``` this will generate kubectl pg plugin's executable in ```go/bin/kubectl-pg.exe```
+
+### To list all commands avaialble in kubectl pg
+
+```kubectl pg --help``` (or) ```kubectl pg```
+
+### To validate postgresql CRD creation
+
+```kubectl pg check```
+
+### To create postgresql cluster using manifest file
+
+```kubectl pg create -f acid-minimal-cluster.yaml```
+
+### To update existing cluster using manifest file
+
+```kubectl pg update -f acid-minimal-cluster.yaml```
+
+### To delete existing cluster using manifest file
+
+```kubectl pg delete -f acid-minimal-cluster.yaml```
+
+### To delete existing cluster using cluster name
+
+```kubectl pg delete acid-minimal-cluster```
+
+--namespace or -n flag to specify namespace if cluster is in other namespace.
+
+```kubectl pg delete acid-minimal-cluster -n namespace01```
+
+### To list postgres resources for the current namespace
+
+```kubectl pg list```
+
+### To list postgres resources across namespaces
+
+```kubectl pg list all```
+
+### To add-user and roles to existing cluster
+
+```kubectl pg add-user USER01 -p CREATE,UPDATE,DELETE -c acid-minimal-cluster```
```

Comment:
This adds users to the manifest, right? I think then the privileges can only be of: ```[SUPERUSER, REPLICATION, INHERIT, LOGIN, NOLOGIN, CREATEROLE, CREATEDB, BYPASSURL]```

Reply:
yes, as I do not know the set of privileges, I added it that way will change it.

Reply:
@VineethReddy02 
pls note by convention the operator does (from [cluster manifest reference](https://github.com/zalando/postgres-operator/blob/master/docs/reference/cluster_manifest.md)):
 > A login user is created by default unless NOLOGIN is specified, in which case the operator creates a role. 

Reference: https://www.postgresql.org/docs/11/role-attributes.html

Reply:
@felix as a user can be created without any privileges does plugin needs to add any privileges if it's nil?